### PR TITLE
Lesson8

### PR DESCRIPTION
**Pull request by ameyunaikgithub**
i have kept my commented line of code for reference . I have implemented only displaying video on browser and just clicking a picture . -- as per lesson instruction...

### CODE REVIEW

File: webcam/public/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
```
Comment (saumya665) [inline webcam/public/index.html]: nothing included in title?

File: webcam/public/index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  </head>
+  <body>
+    Name:<input type="text" id="Name" />
+    Comment:<input type="text" id="Comment" />
+  <!--<input type="submit" id="button" />-->
+    Picture:<input type="submit" value="take picture" id = "click"/>
```
Comment (saumya665) [inline webcam/public/index.html:12]: why is value included here?

File: webcam/public/index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      const content = document.getElementById('chatid');
+      content.innerHTML=xhttp.responseText;
+      console.log(content.innerHTML);
+    }
+  }
+  xhttp.send();
+}
+*/
+/*Camera*/
+const video=document.getElementById('v1')
```
Comment (saumya665) [inline webcam/public/index.html]: missing semicolon at end of line

File: webcam/public/index.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  const comment=document.getElementById("Comment").value;
+  const xhttp=new XMLHttpRequest();
+  xhttp.open('POST', '/newchat');
+  xhttp.setRequestHeader('content-type', 'application/json');
+  xhttp.send(JSON.stringify(
+        {name, comment}
+  )
+  );
+}
+const startchat=()=>{
```
Comment (saumya665) [inline webcam/public/index.html:30]: why is chatbox commented out?

File: webcam/public/index.html (hunk 2 above)
Comment (ameyunaikgithub) [reply webcam/public/index.html:12]: to change my text of button to take picture. i guess value is the text what you want to print on the button

File: webcam/public/index.html (hunk 4 above)
Comment (ameyunaikgithub) [reply webcam/public/index.html:30]: lesson 8 tells you to only "Get access to webcam. When user clicks a button, take a picture by drawing it to canvas" i dont think so chatbox is required in this task

File: webcam/public/index.html (hunk 3 above)
Comment (ameyunaikgithub) [reply webcam/public/index.html]: thanks. will do that

**Review by saumya665 — APPROVED** [review]

File: webcam/public/index.html (hunk 2 above)
Comment (seemcat) [reply webcam/public/index.html:12]: Dude you have two index.html

**Review by seemcat — CHANGES_REQUESTED** [review]

**Review by MadhuBeesetty — APPROVED** [review]

**Pull request merged**